src/screens/FeedScreen/Header.js

```diff
@@ -603,7 +603,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color for the style has been updated. Check if this change is in line with your design guidelines.
```diff
- color: COLORS.blackgrey
+ color: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->